src/App.css

```diff
@@ -1,51 +1,109 @@
 body {
   margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: 'Montserrat', sans-serif;
+}
+
+.header {
+  text-transform: uppercase;
+  margin: 8rem 0 0 2rem;
+  font-size: 2rem;
+  color: #888;
+  font-weight: bold;
+}
+
+button {
+  padding: 1rem;
+  background-color: #0290ff;
+  border-radius: 4px;
+  width: 10rem;
+  height: 3rem;
+  color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: larger;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] Please remove the border of your buttons :wink: to match the original design